Doc/c-api/code.rst

```diff
@@ -51,3 +51,11 @@ bound into a function.
    Return a new empty code object with the specified filename,
    function name, and first line number.  It is illegal to
    :func:`exec` or :func:`eval` the resulting code object.
+
+.. c:function:: int PyCode_Addr2Line(PyCodeObject *co, int byte_offset)
+
+    Return the line number of the of instruction that occurs on or before ``byte_offset`` and ends after it.
+    If you just need the line number of a frame, use :c:func:`PyFrame_GetLineNumber` instead.
+
+    For effciently determining line numbers in a code object, use `the API described in PEP 626
+    <https://www.python.org/dev/peps/pep-0626/#out-of-process-debuggers-and-profilers>`_.
```

Comment:
I don't understand. PyLineTable_InitAddressRange() doesn't seem to be implemented in Python 3.10. Do you plan to implement it?

If it's not implemented yet, I suggest to remove this paragraph, and add it once it will be implemented.

Reply:
It is implemented https://github.com/python/cpython/blob/master/Objects/codeobject.c#L1259
It isn't part of the API, though. Which is why we need to keep `PyCode_Addr2Line`